backend/app/repository/gitee_provider.py

```
@@ -0,0 +1,737 @@
+                        full_name=repo["full_name"],
+                        clone_url=repo["clone_url"],
+                        git_domain=git_domain,
+                        type="gitee",
+                        private=repo["private"]
+                    ).model_dump() for repo in paginated_repos
+                ])
+                continue
+
+            try:
+                # Gitee uses access_token as query parameter
+                response = requests.get(
```

Ruff (0.14.4)

Probable use of `requests` call without timeout

(S113)